app/assets/javascripts/templates/price_breakdown.html.haml

```diff
@@ -1,6 +1,6 @@
-.joyride-tip-guide.price_breakdown{"ng-class" => "{ in: tt_isOpen, fade: tt_animation }"}
+.joyride-tip-guide.price_breakdown{ng: {class: "{ in: tt_isOpen, fade: tt_animation }", show: "tt_isOpen"}}
   %span.joyride-nub.top
-  .background{ng: {show: "tt_isOpen", click: "tt_isOpen = false"}}
+  .background{ng: {click: "tt_isOpen = false"}}
```

Comment:
is there a way we can run rubocop for haml?
I notice here that we dont have the normal spaces inside {} but we have on the line above after tt_animation

Reply:
Not that I know of.

The spaces above are AngularJS syntax while the rest here is HAML syntax. I prefer no spaces within the brackets as in the [HAML reference](https://haml.info/docs/yardoc/file.REFERENCE.html) and in Ruby code.

Angular is different though. Looking at https://angularjs.org/ I see no spaces for single variable names but spaces for expressions with filters.

Reply:
the ruby style is with spaces, right?
https://github.com/rubocop-hq/ruby-style-guide#spaces-braces

Reply:
Only for blocks. Hashes can be written in both ways according to that style guide. But rubocop seems to default to spaces here. I didn't know that and always preferred no spaces. The main thing is consistency, so ... okay, I should start using spaces in hash literals and HAML...

Reply:
yeah, I thought rubocop was demanding spaces everywhere.